src/include/sof/audio/kpb.h

```diff
@@ -61,6 +61,7 @@ enum kpb_state {
 	KPB_STATE_PREPARING,
 	KPB_STATE_RUN,
 	KPB_STATE_BUFFERING,
+	KPB_STATE_INIT_DRAINING,
```

Comment:
and we can probably have a big fat comment here that any state changes must also be reflected in ASCII header diagram or UML diagram on sof-docs (with a link to said UML diagram).

Reply:
Yes very good idea, so let me first finish UML merging and once this is done I will come back here with the update.

Reply:
Please use doxygen and insert link properly, check here: https://stackoverflow.com/a/54637991